cmd/csi-attacher/main.go

```diff
@@ -114,18 +114,17 @@ func main() {
 		}
 		glog.V(2).Infof("CSI driver name: %q", attacher)
 
-		// Check it's ready
```

Comment:
Now I see what you meant. Can you move this check before `ctx, cancel := context.WithTimeout(context.Background(), csiTimeout)` ~10 lines above?

I think the order of `waitForDriverReady()` and `GetDriverName()` does not really matter, so let's do probe first and then start 1 context for GetDriverName+SupportsPluginControllerService+SupportsControllerPublish.

Reply:
Hi, @jsafrane 
Currently, I move the probe check before GetDriverName+SupportsPluginControllerService+SupportsControllerPublish and increase probe timeout. Would you please review this PR again?